packages/theme-dark-extension/style/variables.css

```diff
@@ -374,4 +374,15 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-search-toggle-off-opacity: 0.5;
   --jp-search-toggle-hover-opacity: 0.75;
   --jp-search-toggle-on-opacity: 1;
+
+  /* scrollbar related styles. Supports every browser except Edge.
+   * On Firefox, only the color settings are respected.
+   */
+
+  --jp-scrollbar-background-color: #3f4244;
+  --jp-scrollbar-endpad: 3px; /* the minimum gap between the thumb and the ends of a scrollbar */
+
+  --jp-scrollbar-thumb-color: 88, 96, 97; /* need to specify thumb color as an RGB triplet */
```

Comment:
Same here about adding note about where the color comes from.

Reply:
Same

Reply:
note added